### DISCUSSION TITLE
Ubuntu Installer - cannot find select-7z-arch.js

### DISCUSSION THREAD

**Discussion by willsliou** [Chia-Network/chia-blockchain#4223] [Support]
Hi,
Upon running ./install-gui.sh, I run into the error "cannot find select-7z-arch.js". I'm running the commands as root with sudo-i. Did anyone else run into this error? 

![Screenshot from 2021-05-08 15-39-06](https://user-images.githubusercontent.com/27534106/117551481-37fb9080-b014-11eb-8ee9-a28b0e55c92d.png)
![Screenshot from 2021-05-08 15-43-13](https://user-images.githubusercontent.com/27534106/117551484-3af68100-b014-11eb-83b5-cc3475a0c4c0.png)

**Comment by manasmahodaya** [comment]
yes, i do not have the exact error but this is what i got on my ubuntu 16.04

[11638:0509/143700.146246:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/samwatt/chia-blockchain/chia-blockchain-gui/node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.
/home/samwatt/chia-blockchain/chia-blockchain-gui/node_modules/electron/dist/electron exited with signal SIGTRAP
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chia-blockchain@ electron: `electron .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chia-blockchain@ electron script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/samwatt/.npm/_logs/2021-05-09T09_07_02_118Z-debug.log

**Comment by rcamarda390** [comment]
I got the same error with 1.1.4. I gave up.
I am using RPI4 with 4gb

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
